app/controllers/api/v1/open_library_controller.rb

```diff
@@ -0,0 +1,10 @@
+module Api
+  module V1
+    class OpenLibraryController < ApplicationController
+      def show
+        book = OpenLibraryService.new(params[:isbn]).book_info
+        render json: book
```

Comment:
manage status code when book not found
```suggestion
        book_info = OpenLibraryService.new(params[:isbn]).book_info
        render json: book_info.response, status: book_info.code
```

Reply:
this is necessary? because by making only this change [https://github.com/wx-inc-trainings/np-rails/pull/4#discussion_r640748519](url) would be generating a response:

![Captura de Pantalla 2021-05-27 a la(s) 13 03 39](https://user-images.githubusercontent.com/52846611/119859780-0e83a580-beec-11eb-8514-8759302b961e.png)

Reply:
by default, render response status 200 not necessary include inside the JSON object https://apidock.com/rails/ActionController/Base/render

Reply:
then in the controller there would be only this:

`render json: book_info`

and in the service this:

`{ response: build_response(hash) }`